local_server/telemetry_server/security.py

```
@@ -0,0 +1,87 @@
+from __future__ import annotations
+
+import hashlib
+import hmac
+import time
+from collections import defaultdict
+from threading import Lock
+
+# ---------------------------------------------------------------------------
+# ★ 与客户端 token_tracker.py 中的 _TELEMETRY_HMAC_SECRET 保持一致
+# ---------------------------------------------------------------------------
+DEFAULT_HMAC_SECRET = "neko-v1-a3f8b2c1d4e5f6789012345678abcdef"
```

Ruff (0.15.5)

[error] Possible hardcoded password assigned to: "DEFAULT_HMAC_SECRET"

(S105)